README-Mandrel.md

```diff
@@ -35,16 +35,19 @@ Mandrel distributions can be downloaded from [the repository's releases](https:/
 Mandrel's `native-image` depends on the following packages:
 * glibc-devel
 * zlib-devel
+* freetype-devel
 * gcc
 
 On Fedora/CentOS/RHEL they can be installed with:
 ```bash
-dnf install glibc-devel zlib-devel gcc
+dnf install glibc-devel zlib-devel gcc freetype-devel libstdc++-static
 ```
 
+Note the package might be called `glibc-static` instead of `libstdc++-static`.
+
 On Ubuntu-like systems with:
 ```bash
-apt install libc6-dev zlib1g-dev gcc
+apt install gcc zlib1g-dev build-essential libfreetype6-dev
```

Comment:
isn't build-essential a bit aggressive/bloated?

Reply:
@zakkak I had an issue with static glibc and this did the trick. I am by no means an Ubuntu user though, so feel free to suggest something different that would work on vanilla Ubuntu 18 and 20.

Reply:
Suggestion added in https://github.com/graalvm/mandrel-packaging/pull/96#discussion_r564465432 let's keep the discussion there and apply the changes everywhere later :)